packages/graphql/pubspec.yaml

```diff
@@ -32,4 +32,4 @@ dev_dependencies:
   lints: ^1.0.1
 
 environment:
-  sdk: '>=2.15.0 <4.0.0'
+  sdk: '>=3.3.0 <4.0.0'
```

Comment:
why we need 3.3? and not just 3.0?

Reply:
I downgraded to 3.0.0 and ran `make check`. Tests still passed.